.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -32,60 +18,18 @@ body:
           required: true
         - label: I searched the issue tracker but was unable to find my issue.
           required: true
-        - label: I read the [extended documentation in general](https://docker-mailserver.github.io/docker-mailserver/latest/) but found nothing to resolve the issue.
-          required: true
-        - label: I read the [documentation on debugging](https://docker-mailserver.github.io/docker-mailserver/latest/config/debugging/), tried the proposed steps to debug the problem, but was still unable to resolve the issue.
-          required: true
-        - label: I have read this project's [Code of Conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md) and I agree
+        - label: |
+            I read
+
+              - the [extended documentation in general](https://docker-mailserver.github.io/docker-mailserver/latest/) but found nothing to resolve the issue;
+              - the [documentation on debugging](https://docker-mailserver.github.io/docker-mailserver/latest/config/debugging/), tried the proposed steps to debug the problem, but was still unable to resolve the issue;
+              - this project's [Code of Conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md) and I agree;
+              - the [documentation about filing a bug report](https://docker-mailserver.github.io/docker-mailserver/latest/contributing/issues-and-pull-requests/#filing-a-bug-report).
```

Comment:
![who actually does that? :P](http://25.media.tumblr.com/tumblr_m5ztiwfwQO1rr3l61o1_500.gif)

This will just get checked regardless, but here's hoping some at least gain awareness through the links like debugging docs :+1:

Reply:
😆 Absolutely right! We did our part though 👍🏼